utils.py

```diff
@@ -128,8 +140,10 @@ def add_or_update_ee_layer(eeObject, visParams, name, shown, opacity):
         except:
             name = "untitled"
 
-    layer = add_or_update_ee_image_layer(image, name, shown, opacity)
+    image.ee_instance = type(eeObject)
```

Comment:
Just a naming comment: ee_instance implies an object, but this is still a type, just a qgis_ee_type or a-like.

Reply:
you're right, fixed in a8873156f89c8a6fe9f8d111173c1616b7c7a259